components/AtlasController/index.js

```diff
@@ -173,7 +189,23 @@ const AtlasController = ({ width, height }) => {
             setHoverSSID(null);
           }
         }}
+        bboxHandler={setMapBBox}
       />
+      <HStack
+        pos="absolute"
+        top="105px"
+        right="55px"
+        zIndex={9}
+        bgColor="white"
+        p={2}
+        borderRadius={4}
+        boxShadow="sm"
+        onClick={() => setSearchMove(!searchMove)}
+        cursor="pointer"
+      >
+        <Checkbox isChecked={searchMove} />
```

Comment:
There's some issue with the checkbox CSS. Clicking inside of it does nothing, but clicking around it checks/unchecks it.